sql/projects_fisa.sql

```diff
@@ -50,8 +50,16 @@ CREATE TABLE cpdb_projects AS(
                c.nccother,
                c.noncitycost,
                c.totalcost,
-               a.cape_agencyacronym as magencyacro,
-               a.cape_agency as magencyname
+               (CASE
+               WHEN upper(s.description) LIKE '%BNY%' AND s.magency = '801' THEN 'BNY'
+               WHEN lower(s.description) LIKE '%governor%s island%' AND s.magency = '801' THEN 'TGI'
+               ELSE a.cape_agencyacronym
+               END) as magencyacro,
```

Comment:
When description doesn't have BNY nor governors island in description then takes correctly value based on matching agency id. Used this sql query to check
```
select * from cpdb_projects where magencyname LIKE '%Education%';
```
Matched on agency id 040